examples/fishjam-chat/package.json

```diff
@@ -8,7 +8,7 @@
     "android": "expo run:android",
     "ios": "expo run:ios",
     "test": "jest",
-    "build": "tsc",
+    "build": "tsc && tsc -p ./plugin",
```

Comment:
What does `-p` options does? I can't find it in [official docs](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

Reply:
It's the same as `--project`. I'm surprised it's not mentioned in the docs however it's there when you run `tsc --help`.

Reply:
Hmm, maybe lets use `--project` then? 🤔 
(or leave as is, no strong opinion)

Reply:
Sure!